.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "docker.host":"tcp://51.140.183.174:2376",
```

Comment:
@js947 Looks like this sneaked into a commit (along with tasks.json). Otherwise this looks good to me

Reply:
oops that was dumb. Thanks for pointing that out, now fixed and erased from the diff...